spec/services/scrapers/low_priority_spec.rb

```diff
@@ -4,16 +4,29 @@
 
 RSpec.describe Scrapers::LowPriority do
   describe '#perform' do
-    it 'add specs' do
+    it 'adds jobs to the CourtCaseWorker' do
       test_case = create(:court_case, :with_html, closed_on: 7.months.ago)
       test_case.case_html.update(scraped_at: 7.months.ago)
+      data = nil
 
-      data = described_class.perform
+      expect do
+        data = described_class.perform
+      end.to change(CourtCaseWorker.jobs, :size).by(1)
 
       expect do
         described_class.perform
-      end.to change(CourtCaseWorker.jobs, :size).by(1)
+      end.to change(CourtCaseWorker.jobs, :size).by(0)
+
       expect(data.first.id).to eq test_case.id
     end
+    it 'doesnt add duplicates to worker' do
+      test_case = create(:court_case, :with_html, closed_on: 7.months.ago)
+      test_case.case_html.update(scraped_at: 7.months.ago)
```

Comment:
these should be in a `before` block (try to avoid doing setup in the `it` block)